### PR TITLE
Admin Permissions:

### PR DESCRIPTION
**Pull request by GalinaRuban**
test createPermissionsForAdminScreensTest as per https://trello.com/c/ejq5qyuq/840-admin-permission-settings-for-admin-screens (only checkboxes are being verified as of now, verifying with George how functionality can be checked for admin) + permission folder with two subfolders (admin and user) were created and all permission admin tests were combined into one class. Permissions model part will be changed/adjusted later on.

### CODE REVIEW

File: src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public AdminMenuPermissionsPage selectCheckBox() {
         return new AdminMenuPermissionsPage(getDriver());
     }
 
-    public boolean isCheckBoxSelected() {
+    public boolean isUserCheckBoxSelected() {
```
Comment (oooleks) [inline src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java:44]: Redundant `if `statement, could be just: `return userMenusCheckBox.isSelected();`

File: src/test/java/model/admin/configuration/permissions/AdminScreenPermissionsPage.java [hunk 2]
```diff
@@ -36,4 +38,31 @@ public boolean isUserCheckBoxSelected() {
         }
         return false;
     }
-}
\ No newline at end of file
+
+    public boolean isAdminCheckBoxSelected() {
+        if (adminHomeScreenCheckBox.isSelected()) {
```
Comment (oooleks) [inline src/test/java/model/admin/configuration/permissions/AdminScreenPermissionsPage.java:43]: Redundant `if `statement

File: src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java (hunk 1 above)
Comment (GalinaRuban) [reply src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java:44]: да, оно мне говорило что if statement can be simplified, надо было слушать)))

File: src/test/java/test/admin/permissions/forAdmin/AdminPermissionsForAdminTest.java [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                .isAdminCheckBoxSelected());
+    }
+
+    @Test(dependsOnMethods = "createPermissionsForAdminScreensTest")
+    public void withdrawPermissionsForAdminScreensTest() {
+
+        getDriver().get(instanceURL);
+
+        ConfigurationPage configurationPage = new ConfigurationPage(getDriver());
+        configurationPage
```
Comment (oooleks) [inline src/test/java/test/admin/permissions/forAdmin/AdminPermissionsForAdminTest.java:134]: #134 line could be deleted

File: src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java (hunk 1 above)
Comment (GalinaRuban) [reply src/test/java/model/admin/configuration/permissions/AdminMenuPermissionsPage.java:44]: will be done with the next commit

**Review by guram21 — APPROVED** [review]

**Review by oooleks — APPROVED** [review]

**Pull request merged**